js/utils/validateMediumEditor.js

```diff
@@ -6,7 +6,7 @@ function checkVal($field) {
     $field.val('');
   }
 
-  //replace double quotes with single quotes to avoid invalid json
+  // replace double quotes with single quotes to avoid invalid json
   fVal = fVal.replace(/\\([\s\S])|(")/g, "'");
```

Comment:
It was in there to stop the sanitize from breaking the json, if that's fixed it can be taken out. I'll patch it later.